src/sidebar/store/test/use-store-test.js

```diff
@@ -16,90 +18,262 @@ const reducer = (state = initialState, action) => {
   }
 };
 
-describe('useStore', () => {
-  let renderCount;
-  let testStore;
-  let TestComponent;
-
-  beforeEach(() => {
-    renderCount = 0;
-
-    // eslint-disable-next-line react/display-name
-    TestComponent = () => {
-      renderCount += 1;
-      const aValue = useStore(store => store.getState().value);
-      return <div>{aValue}</div>;
-    };
-
-    testStore = createStore(reducer);
-    $imports.$mock({
-      '../util/service-context': {
-        useService: name => (name === 'store' ? testStore : null),
-      },
-    });
-  });
+// Store module for use with `createStore` in tests.
```

Comment:
Since this hook is intended to be used with a store created by _our_ `createStore` function (as opposed to the one from the `redux` package) I figured it was better to use such a store in the tests.